plerkle/src/geyser_plugin_nft.rs

```diff
@@ -345,10 +349,10 @@ impl GeyserPlugin for Plerkle<'static> {
                 msg.add_stream(SLOT_STREAM).await;
                 msg.add_stream(TRANSACTION_STREAM).await;
                 msg.add_stream(BLOCK_STREAM).await;
-                msg.set_buffer_size(ACCOUNT_STREAM, 100_000_000).await;
-                msg.set_buffer_size(SLOT_STREAM, 100_000).await;
-                msg.set_buffer_size(TRANSACTION_STREAM, 10_000_000).await;
-                msg.set_buffer_size(BLOCK_STREAM, 100_000).await;
+                msg.set_buffer_size(ACCOUNT_STREAM, config.account_stream_size.unwrap_or(100_000_000)).await;
```

Comment:
I think we should bump this to 250mn at least. 100mn will be too little for the token accounts.